keras_cv/models/object_detection/yolo_v8/yolo_v8_detector_test.py

```diff
@@ -171,21 +191,25 @@ def test_update_prediction_decoder(self):
             backbone=keras_cv.models.YOLOV8Backbone.from_preset(
                 "yolo_v8_s_backbone"
             ),
-            prediction_decoder=keras_cv.layers.MultiClassNonMaxSuppression(
+            prediction_decoder=keras_cv.layers.NonMaxSuppression(
                 bounding_box_format="xywh",
                 from_logits=False,
                 confidence_threshold=0.0,
                 iou_threshold=1.0,
             ),
         )
 
-        image = tf.ones((1, 512, 512, 3))
+        image = np.ones((1, 512, 512, 3))
 
         outputs = yolo.predict(image)
         # We predicted at least 1 box with confidence_threshold 0
-        self.assertGreater(outputs["boxes"]._values.shape[0], 0)
+        self.assertGreater(outputs["boxes"].shape[0], 0)
 
-        yolo.prediction_decoder = keras_cv.layers.MultiClassNonMaxSuppression(
+        # TODO(tirthasheshpatel): Updating any model attribute doesn't work
+        # with Keras Core once the model is build.
+        if multi_backend():
+            pytest.xfail("Keras Core doesn't allow updating attributes.")
```

Comment:
@ianstenbit Note that I turned off this test since it failed for all the backends. Looks like, unlike Keras, Keras-Core doesn't retrace the graph on attribute update. I think it's OK if we add support for it later on, what do you think?